backend/src/v5/middleware/dataConverter/multer.js

```diff
@@ -61,7 +61,7 @@ const imageFilter = (req, file, cb) => {
 const ensureFileIsImage = async (req, res, next) => {
 	const fileBuffer = req.file?.buffer;
 	if (fileBuffer) {
-		const type = await FileType.fromBuffer(fileBuffer);
+		const type = await fileTypeFromBuffer(fileBuffer);
```

Comment:
Can we extrapolate this a bit more

maybe split it into 2 functions

one that returns the extension, the other returns the MIME type

Reply:
something like this ?
![image](https://user-images.githubusercontent.com/71635617/155318137-bda7d59a-0c59-4618-ba8d-f2cdc32d0c81.png)

Reply:
sure, i don't think you need the top one though

You just need another function that returns you the mime type